libs/atqr/domain/src/lib/domain-spec/stories/verifyIfChallengeCanBegin.disl

```diff
@@ -1,6 +1,9 @@
+context Chalennge
+
 entity Challenge
 
   💬 verifyIfChallengeCanBegin!
```

Comment:
is not this suppose to be a function instead of a notification ?

Reply:
Yep!